From 3f6fa77c67210d506cb952ae59dba2b2ddc79e0a Mon Sep 17 00:00:00 2001
From: Claudiu Manoil <claudiu.manoil@nxp.com>
Date: Fri, 23 Feb 2018 14:37:47 +0200
Subject: [PATCH 112/706] enetc: Update PF and VF driver versions to v0.7
 (m0169)

Updated drivers to BG v88 (same as v86), for simulator model m0169.
Includes:
* VLAN insertion and extended Tx BD support
* Reworked MAC addr filtering to conform BG v88
* configurable BD VLAN extraction (Rx VLAN no longer fixed as "on")
* MAC counters available via ethtool -S
* v88 register and other updates
* driver improvements and fixes

Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
(cherry picked from commit d31955e9e21235455f599f9b61787f914d3fd825)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/net/ethernet/freescale/enetc/enetc_pf.c | 2 +-
 drivers/net/ethernet/freescale/enetc/enetc_vf.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/freescale/enetc/enetc_pf.c b/drivers/net/ethernet/freescale/enetc/enetc_pf.c
index cad80cc87499..65627b1fcea2 100644
--- a/drivers/net/ethernet/freescale/enetc/enetc_pf.c
+++ b/drivers/net/ethernet/freescale/enetc/enetc_pf.c
@@ -36,7 +36,7 @@
 #include "enetc_pf.h"
 
 #define ENETC_DRV_VER_MAJ 0
-#define ENETC_DRV_VER_MIN 6
+#define ENETC_DRV_VER_MIN 7
 
 #define ENETC_DRV_VER_STR __stringify(ENETC_DRV_VER_MAJ) "." \
 			  __stringify(ENETC_DRV_VER_MIN)
diff --git a/drivers/net/ethernet/freescale/enetc/enetc_vf.c b/drivers/net/ethernet/freescale/enetc/enetc_vf.c
index 8046c562bcf2..c078644665fa 100644
--- a/drivers/net/ethernet/freescale/enetc/enetc_vf.c
+++ b/drivers/net/ethernet/freescale/enetc/enetc_vf.c
@@ -36,7 +36,7 @@
 #include "enetc.h"
 
 #define ENETC_DRV_VER_MAJ 0
-#define ENETC_DRV_VER_MIN 6
+#define ENETC_DRV_VER_MIN 7
 
 #define ENETC_DRV_VER_STR __stringify(ENETC_DRV_VER_MAJ) "." \
 			  __stringify(ENETC_DRV_VER_MIN)
-- 
2.17.1

